lib/bolt/node/winrm.rb

```diff
@@ -297,8 +323,22 @@ def _run_task(task, input_method, arguments)
         Bolt::Node::Success.new
       end.then do
         with_remote_file(task) do |remote_path|
-          path, args = *process_from_extension(remote_path)
-          execute_process(path, args, stdin)
+          if powershell_file?(remote_path) && stdin.nil?
+            execute(<<-PS)
+try
+{
+  # TODO: powershell doesn't support STDIN redirection when executing a script
```

Comment:
I don't think that we care about this `TODO`, but should discuss / remove the note prior to merging this PR.

Reply:
This seems fine, let's remove the TODO.